lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -231,7 +231,7 @@ export class UserAgentApplication {
    * @param {Array.<string>} scopes - Permissions you want included in the access token. Not all scopes are guaranteed to be included in the access token returned.
    * @param {string} extraQueryParameters - Key-value pairs to pass to the authentication server during the interactive authentication flow.
    */
-  loginRedirect(request: AuthenticationParameters): void {
+  loginRedirect(request?: AuthenticationParameters): void {
```

Comment:
is this nullable?

Reply:
It is! All the params in request are optional. This is because we enabled users to call loginRedirect() and loginPopup() without any arguments when they just need id_token with no specific scopes attached. I just missed making the main argument optional, hence the correction.